packages/govern-server/src/core/data/TheGraphWrapper.ts

```diff
@@ -0,0 +1,70 @@
+import fetch from 'isomorphic-unfetch'
+import { Client } from '@urql/core'
+import { DocumentNode } from 'graphql'
+import { ErrorConnection } from '../errors'
+import { QueryResult } from '../types'
+
+type TheGraphWrapperOptions = {
+  verbose?: boolean
+}
+
+export default class TheGraphWrapper {
+  #client: Client
+  #verbose: boolean
+
+  constructor(
```

Comment:
Do we normally not add function documentations? Could be interesting later to generate a class reference :-)